test/Microsoft.ML.TestFramework/TestCommandBase.cs

```diff
@@ -1271,7 +1271,7 @@ public void CommandTrainTestMoreGains()
             var dataPath = CreateOutputPath("Traindata.txt");
             File.WriteAllLines(dataPath.Path, new string[] {
                 "0,0,1,2", "3,0,4,5", "4,0,5,6", "0,1,1,2", "3,1,4,5", "5,1,5,6" });
-            const string customGains = "0,3,7,15,35,63";
+            double[] customGains = new double[] { 0, 3, 7, 15, 35, 63 };
```

Comment:
>customGains [](start = 21, length = 11)

This is only used to format the string in `extraArgs` below, so it should stay a string.